emp/emp_role_cftemplate.yaml

```diff
@@ -301,23 +301,43 @@ Resources:
               - !Sub 'arn:aws:iam::${AWS::AccountId}:role/${EKSRole}'
             Effect: Allow
           - Action:
-              # these are api calls our controllers make, we should be able to restrict these once we have tagging.
-              # the tagging permissions are left as-is for that purpose
+              # read-only
+              - elasticfilesystem:DescribeTags
+              - elasticfilesystem:ListTagsForResource
+            Resource:
+              - '*'
+            Effect: Allow
+          - Action:
               - elasticfilesystem:CreateFileSystem
+            Resource: '*'
+            Effect: Allow
+            Condition:
+              StringEquals:
+                aws:RequestTag/emp.pf9.io: owned
+              StringLike:
+                aws:RequestTag/emp.pf9.io/baremetalpool: '*'
+                aws:RequestTag/emp.pf9.io/namespace: '*'
+          - Action:
+              - elasticfilesystem:DescribeFileSystems
               - elasticfilesystem:CreateMountTarget
-              - elasticfilesystem:CreateTags
```

Comment:
We are going to add tags to EFS from controllers I believe? Doesn't it require this permission of createTags for an EFS?

Reply:
It is there

Reply:
See: https://github.com/platform9/support-locker/pull/54/commits/e31ca14f4b395aa55601ef075d1294c157478382

Reply:
As discussed, so this permission is not needed specifically, but we do have the permissions to add tags cc: @Rohitrajak1807

Reply:
yes. it's here: https://github.com/platform9/support-locker/pull/54/files#r1773132869